### ISSUE TITLE
`get_manta_sv` is used in `annotate_igh_breakpoints` examples

### ISSUE THREAD

**Issue by vladimirsouza** [morinlab/GAMBLR.utils#5]
`get_manta_sv` is a function restricted for GSC users (from GAMBLR.results package). I think we could keep this example, but it would be great if we also include another example for non-GSC users.

Current example:
```
manta_sv = get_manta_sv(verbose = FALSE)
all_annotated = annotate_sv(sv_data = manta_sv)
ig_annotated = annotate_igh_breakpoints(all_annotated)
```

**Comment by rdmorin** [comment]
Does the bundled data have Manta SVs in it? If so, we could use that as a source of input for a generic example.

**Comment by Kdreval** [comment]
Yes, bundled data in GAMBLR.data has manta SVs and they can be used in this example

**Comment by mattssca** [comment]
For reference, these would be the datasets we're discussing here:

`GAMBLR.data::sample_data$grch37$bedpe`
`GAMBLR.data::sample_data$hg38$bedpe`

**Comment by vladimirsouza** [comment]
The issue was addressed in PR [#7](https://github.com/morinlab/GAMBLR.utils/pull/7).
